Audacia.Log.AspNetCore/README.md

```diff
@@ -96,11 +96,42 @@ public IConfiguration Configuration { get; set; }
 public void ConfigureServices(IServiceCollection services)
 {
   services.ConfigureActionContentLogging(Configuration);
-  services.AddRequestBodyTelemetry();
+  services.AddActionRequestBodyTelemetry(Configuration);
   services.AddControllers(x => x.Filters.Add<LogRequestBodyActionFilterAttribute>());
 }
 ```
 
+#### ResponseBodyActionLogFilter
+This filter can be registered to include logs for the beginning and end of each Action Method. This only includes response body as well as details of the response such as the type of model returned. Register it like so:

```

Comment:
Appreciate this is a copy paste and slight tweak to the description above, but not really sure this describes what either of these filters do very well.

It doesn't add logs for the beginning and end of each action method? Could say "This filter can be registered to include the response body to each action" or something? Can change the the one for request body too?

Reply:
Yeah good point I'll modify it and we can review to see if it is more fitting?

Reply:
All the docs look great, thanks Rhys